fastplotlib/graphics/selectors/_linear.py

```diff
@@ -245,7 +255,7 @@ def _ipywidget_callback(self, change):
         if self._block_ipywidget_call:
             return
 
-        self.selection = change["new"]
+        self._selection = change["new"]
```

Comment:
once `_selection` is defined in the construction just access it using the property.